### ISSUE TITLE
Совместимость поведения ЧтениеТекста с разными версиями 1С

### ISSUE THREAD

**Issue by Mr-Rm** [EvilBeaver/OneScript#1179]
**Опишите ошибку**
Объект `ЧтениеТекста` и тесты на совместимость для него создавались в 2016 году.
Но с выходом версии 8.3.11 в платформу были внесены изменения:
> Пропущенные параметры РазделительСтрок и КонвертируемыйРазделительСтрок в конструкторе объекта ЧтениеТекста интерпретируются как Символы.ПС и Символы.ПС+Символы.ВК (соответственно).
> Устранено некорректное поведение системы. Значения пропущенных параметров стали соответствовать описанным в Синтакс-помощнике.
Источник: https://dl03.1c.ru/content/Platform/8_3_11_2867/1cv8upd.htm#71ffa53e-0fa9-11e7-a3f7-0050569f678a

Ряд тестов в tests\text-read.os не соответствует современным версиям платформы.

**Ожидаемое поведение**
Привести к актуальным на сегодняшний день версиям.

**Дополнительная информация**
Не исключено, что есть и другие объекты с "устаревшим" поведением.

**Comment by johnnyshut** [comment]
@EvilBeaver посмотришь pr?

**Comment by EvilBeaver** [comment]
@johnnyshut Да, но я очкую насчет совместимости уже с 1Скрипт

**Comment by johnnyshut** [comment]
> [@johnnyshut](https://github.com/johnnyshut) Да, но я очкую насчет совместимости уже с 1Скрипт

всмысле, что где то в самом onescript, что то случится?

**Comment by EvilBeaver** [comment]
> > [@johnnyshut](https://github.com/johnnyshut) Да, но я очкую насчет совместимости уже с 1Скрипт
> 
> всмысле, что где то в самом onescript, что то случится?

Нет, в смысле, что существующие проекты на 1Скрипт отвалятся. Но я вижу, ты уже проверил основные приложения. Спасибо!

**Comment by johnnyshut** [comment]
> > > [@johnnyshut](https://github.com/johnnyshut) Да, но я очкую насчет совместимости уже с 1Скрипт
> > 
> > 
> > всмысле, что где то в самом onescript, что то случится?
> 
> Нет, в смысле, что существующие проекты на 1Скрипт отвалятся. Но я вижу, ты уже проверил основные приложения. Спасибо!

Скачал все библиотеки и проверил, нашлась гиппотетическая поломка в `v8runner`. Проверил  `v8runner` на реальном пути чтения (`ОтключитьОтХранилищаВсеРасширения`) и дополнительно прогнал сравнение старого/нового парсинга переносов (`CRLF`) - поведение не меняется и поломки не воспроизводятся.
